README.md

```diff
@@ -1,36 +1,72 @@
-#serverless-plugin-localstack
+# serverless-localstack
 [Serverless](https://serverless.com/) Plugin to support running against [Atlassian Labs Localstack](https://github.com/atlassian/localstack).
 
+This plugin allows any requests to AWS to be redirected to a running Localstack instance.
+
 WARNING: This plugin is very much WIP
 
 Pre-requisites:
-* Docker
-* Docker compose
-* Serverless: `npm install -g serverless`
+* Localstack
+
+## Installation
 
+The easiest way to get started is to install via npm.
 
-Getting Started:
+    npm install -g serverless
+    npm install -g serverless-localstack
 
-* Clone the repository
-`git clone https://github.com/temyers/serverless-localstack`
+## Installation (without npm)
 
-* Start localstack
```

Comment:
That's a fair point. I think the reason why I decided to pull this section was due to the many different ways to start up Localstack and it's documentation is decent.